### PR TITLE
add jquery-confirm v2.5.1 with autoupdate config

### LINKED ISSUE #6888: [Request] jquery-confirm
**Library name:** jquery-confirm
**Git repository url:** https://github.com/craftpip/jquery-confirm
**npm package url(optional):** 
**License(s):** MIT License
**Official homepage:** http://craftpip.github.io/jquery-confirm/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by nolsherry**
@PeterDaveHello  
I already finished this pull request, please help me check, thanks! 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.
Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/craftpip/jquery-confirm
 * Official website (optional, not the repository): http://craftpip.github.io/jquery-confirm/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 229
   - Count of watchers: 34
   - Count of forks: 79
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #6888, cc @craftpip

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@nolsherry Pleasr ask other @cdnjs/intern2 members first.

File: ajax/libs/jquery-confirm/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "alert",
+    "dialog"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/craftpip/jquery-confirm.git",
+    "basePath": "dist",
+    "files": [
+      "jquery-confirm*"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-confirm/package.json]: Should be `"**/*"`

**Comment by nolsherry** [comment]
@pvnr0082t  please help me review this pull request, thank you!!!  :)

**Comment by PeterDaveHello** [comment]
@nolsherry you need to fix the problem first

**Comment by nolsherry** [comment]
@PeterDaveHello I have fixed the problem, thanks !

**Comment by PeterDaveHello** [comment]
@nolsherry Call your reviewer first.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this pull request is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Please rebase your branch with the latest master branch every time you modified you commit(s).

**Comment by nolsherry** [comment]
@dannyxx001 Hi please help me to review it. :+1:

**Comment by dannyxx001** [comment]
@PeterDaveHello I think this pull request is ok. Please review it again, thank you.

**Pull request merged**